spec.md

```diff
@@ -718,6 +718,12 @@ For success responses, the following fields are valid.
 }
 ```
 
+If the successful response includes a `state` of `failed` then the Platform
+MUST send a deprovision request to the Service Broker to prevent an orphan
+being created on the Service Broker.  However, while the platform will attempt
```

Comment:
I'd capitalise Platform (multiple occurences in this PR) now to save conflicts when #356 is merged

Reply:
done